mne/event.py

```diff
@@ -190,7 +190,8 @@ def _read_events_fif(fid, tree):
 
 def read_events(filename, include=None, exclude=None, mask=None,
                 mask_type='not_and'):
-    """Read events from fif or text file.
+    """Read events from fif or text file. See :ref:`tut_epoching_and_averaging`
+    as well as :ref:`ex_read_events` for more information about events.
```

Comment:
Descriptions should always be a single line. Extra stuff should be a separate paragraph, so:
```
Read events from fif or text file.

See :ref:`tut_epoching_and_averaging` as well as :ref:`ex_read_events` for more information about events.
```